### PR TITLE
Add libtpu configuration to cpp test script

### PR DESCRIPTION
**Pull request by steventk-g**
**Testing**
`TPUVM_MODE=1 ./test/cpp/run_tests.sh` on TPUVM

### CODE REVIEW

**Review by will-cromar — APPROVED** [review]
LGTM

**Review by JackCaoG — APPROVED** [review]
Thanks!

File: test/cpp/run_tests.sh [hunk 1]
```diff
@@ -47,6 +47,11 @@ if [[ "$TPUVM_MODE" != "1" ]]; then
   export XLA_EXPERIMENTAL
 fi
 
+# We need to configure the libtpu path to test on TPUVM
+if python -c "import libtpu" &> /dev/null; then
```
Comment (vanbasten23) [inline test/cpp/run_tests.sh:51]: curious what does the condition mean

Comment (steventk-g) [reply test/cpp/run_tests.sh:51]: If the import command succeeds i.e. whenever we have libtpu on the machine

**Pull request merged**